README.md

```diff
@@ -39,6 +39,9 @@ mvn verify -DskipTests
 mvn clean verify -Dcucumber.options="--tags @{name of tag, with upper/lowercase letters}"
 ```
 
+## To run OWASP Zap Daemon
+```docker run -u zap -p 8899:8899 -i owasp/zap2docker-stable zap.sh -daemon -host 0.0.0.0 -port 8899 -config api.key=SECRETKEY -config api.addrs.addr.name=.* -config api.addrs.addr.regex=true```
```

Comment:
I'm not sure how readme should be detailed but for less experienced users information about docker installation could be useful (info that docker has to be installed and which images are required)

Reply:
I think the image will be pulled after command execution, about parameters they are from zap proxy itself.

Reply:
You're right!